jobs/ostree-compose.yml

```diff
@@ -71,3 +71,31 @@
 
     publishers:
         - ci-pipeline-duffy-publisher
+
+- job:
+    name: ci-pipeline-ostree-boot-sanity
+    defaults: ci-pipeline-defaults
+    parameters:
+        - string:
+            name: fed_branch
+            description: |
+              which branch to test
+        - string:
+            name: image2boot
+            description: |
+              url pointing to the image to boot
+        - string:
+            name: commit
+            description: |
+              sha pointing to the particular commit we want to test
+    builders:
+        - ci-pipeline-duffy-builder:
+            task: ostree-boot-image
+            variables: |
+                export BUILD="${fed_branch}"
```

Comment:
This is going to give you problems in ostree-boot-image because in that file you do not have a check to convert master to rawhide and you use it in the IMG_URL